app/main.py

```diff
@@ -2,0 +1,22 @@
+from app.cafe import Cafe
+
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict[any]], cafe: Cafe) -> str:
```

Comment:
Code Style: Consider adding a brief docstring to this function to explain what it does, what parameters it expects, and what it returns. This can help other developers understand your code more easily.